packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -61,8 +54,10 @@ const DEFAULT_POST_BASE = {
 		field: 'date',
 		direction: 'desc',
 	},
-	fields: [ 'title', 'author', 'status' ],
-	layout: defaultLayouts[ LAYOUT_LIST ].layout,
+	titleField: 'title',
+	mediaField: 'featured_media',
+	fields: [ 'author', 'status' ],
+	...defaultLayouts[ LAYOUT_LIST ],
```

Comment:
Nice touch removing the `.layout`. This way every property (not only `layout`) will be overriden.